legacy/build-deploy-docker-compose.sh

```diff
@@ -221,8 +229,8 @@ set +x
 currentStepEnd="$(date +"%Y-%m-%d %H:%M:%S")"
 patchBuildStep "${buildStartTime}" "${buildStartTime}" "${currentStepEnd}" "${NAMESPACE}" "initialSetup" "Initial Environment Setup"
 previousStepEnd=${currentStepEnd}
+beginBuildStep "Configured Variables"
```

Comment:
Can we change this to `Configure Variables` and then on line 537, change the `patchBuildStep` to also be the same step name.
Seems weird to be `BEGIN Configured Variables`

Reply:
Yep agreed - updated now